src/Microsoft.DotNet.Interactive.ExtensionLab.Tests/DataFrameKernelExtensionTests.it_registers_formatters.received.json

```diff
@@ -0,0 +1,4 @@
+id           name         color        deliciousness
+1            apple        green        10           
+2            banana       yellow       11           
+3            cherry       red          9000         
```

Comment:
This file shouldn't be part of the PR; it looks like an artifact from an earlier test run.

Reply:
good eyes.